### PR TITLE
Add a README for the collect_signals cmd.

### PR DESCRIPTION
**Pull request by calebbrown**

### CODE REVIEW

File: cmd/collect_signals/README.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+The input of this tool could by the output of the `enumerate_github` tool.
+
+The output of this tool is used as an input for the `scorer` tool, or for input
+into a data analysis system such as BigQuery. 
+
+## Example
+
+```shell
+$ export GITHUB_TOKEN=ghp_x  # Personal Access Token Goes Here 
+$ export ...
```
Comment (oliverchang) [inline cmd/collect_signals/README.md:15]: What is ... referring to? can we omit it for simplicity?

File: cmd/collect_signals/README.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+      GitHub's data too).
+   - Prevents downtime due to local IT failures.
+1. Shard the input repository list and run multiple instances on different
+   hosts with different GitHub tokens:
+    - Ensures work progresses even if one instance stops.
+    - Provides additional compute and network resources.
+
+### Q: How do I restart after a failure?
+
+If running with a single worker this process is fairly straight forward.
```
Comment (oliverchang) [inline cmd/collect_signals/README.md:160]: s/straight forward/straightforward/ ?

File: cmd/collect_signals/README.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+```
+
+#### GCP Authentication
+
+BigQuery access requires the "BigQuery User" (`roles/bigquery.user`) role added
+to the account used, or be an "Owner".
+
+##### Option 1: `gcloud login`
+
+This option is useful during development. Run `gcloud login` to login to GCP.
```
Comment (oliverchang) [inline cmd/collect_signals/README.md:67]: Do we need `--update-adc` here for the client libraries to work?

**Review by oliverchang — APPROVED** [review]
awesome, thanks!!

**Comment by oliverchang** [comment]
Once we have documentation for the enumerate, collect_signals, scorer, could we have a simple example for how to:
1. use all 3 together to generate a list
2. use the collect_signals and scorer to generate the score for a single repo? 

Any other use cases here to cover?

File: cmd/collect_signals/README.md (hunk 1 above)
Comment (calebbrown) [reply cmd/collect_signals/README.md]: This was meant for GCP. Fixed.

File: cmd/collect_signals/README.md (hunk 3 above)
Comment (calebbrown) [reply cmd/collect_signals/README.md]: Added. Thanks.

**Comment by calebbrown** [comment]
> Any other use cases here to cover?

Those 2 are the primary use cases. But generally having a "cookbook" for people to refer to for various ways to use the tools is a very good idea.

**Review by oliverchang — APPROVED** [review]

**Comment by oliverchang** [comment]
Sorry missed that you requested another review here. This LGTM!

**Pull request merged**